web/src/narrow.js

```diff
@@ -420,16 +420,19 @@ export function activate(raw_terms, opts) {
 
         const msg_list = new message_list.MessageList({
             data: msg_data,
-            table_name: "zfilt",
         });
 
         // Show the new set of messages.  It is important to set message_lists.current to
         // the view right as it's being shown, because we rely on message_lists.current
         // being shown for deciding when to condense messages.
         $("body").addClass("narrowed_view");
-        $("#zfilt").addClass("focused-message-list");
-        $("#zhome").removeClass("focused-message-list");
+        msg_list.$list.addClass("focused-message-list");
+        message_lists.home.$list.removeClass("focused-message-list");
```

Comment:
This I guess don't remove `focused-message-list` from other message lists before removing them from the DOM. Not sure whether that's a good thing or a bad thing in terms of how the browser will display the transition; probably it is good or doesn't matter?

Reply:
Yeah, seems like it doesn't matter since we are immediately removing the current list from DOM.